nemo/core/classes/common.py

```diff
@@ -48,6 +54,203 @@
 _HAS_HYDRA = True
 
 
+NEMO_DEFAULT_MODEL_CARD_TEMPLATE = """---
```

Comment:
Can we put this in it's own file?

Reply:
Done, now its under nemo.core.classes.mixins.hf_io_mixin